pkg/config/builder_test.go

```diff
@@ -315,6 +317,8 @@ func TestLoadFile(t *testing.T) {
 		if !reflect.DeepEqual(cfg.rules.rules, test.expected.rules.rules) {
 			t.Errorf("Expected %+v but got %+v", test.expected.rules.rules, cfg.rules.rules)
 		}
+
+		cfg.ShutDown()
```

Comment:
This will not run if the test ends or panics before this point. Can you use `defer cfg.Shutdown()` instead?

Reply:
:+1:

Reply:
Foe some reason defer doesn't work here :-(

Reply:
defer is for functions, it doesn't work in code blocks.